lib/static/new-ui/features/suites/components/TreeActionsToolbar/index.tsx

```diff
@@ -0,0 +1,189 @@
+import {Icon} from '@gravity-ui/uikit';
+import classNames from 'classnames';
+import {
+    SquareCheck,
+    ChevronsExpandVertical,
+    ChevronsCollapseVertical,
+    SquareDashed,
+    Square,
+    CircleInfo,
+    Play,
+    Check,
+    ArrowUturnCcwLeft
+} from '@gravity-ui/icons';
+import React, {ReactNode, useMemo} from 'react';
+import {useDispatch, useSelector} from 'react-redux';
+
+import styles from './index.module.css';
+import {
+    acceptOpened,
+    collapseAll,
+    deselectAll,
+    expandAll,
+    retrySuite,
+    selectAll, staticAccepterStageScreenshot, staticAccepterUnstageScreenshot,
+    undoAcceptImages
+} from '@/static/modules/actions';
+import {ImageEntity, State} from '@/static/new-ui/types/store';
+import {CHECKED, INDETERMINATE} from '@/constants/checked-statuses';
+import {IconButton} from '@/static/new-ui/components/IconButton';
+import {
+    getCheckedTests,
+    getSelectedImages,
+    getVisibleBrowserIds,
+    getVisibleImages
+} from '@/static/modules/selectors/tree';
+import {getIsGui, getIsInitialized, getIsStaticImageAccepterEnabled} from '@/static/new-ui/store/selectors';
+import {isAcceptable, isScreenRevertable} from '@/static/modules/utils';
+import {EditScreensFeature, RunTestsFeature} from '@/constants';
+
+interface TreeActionsToolbarProps {
+    onHighlightCurrentTest?: () => void;
+}
+
+export function TreeActionsToolbar(props: TreeActionsToolbarProps): ReactNode {
+    const dispatch = useDispatch();
+
+    const rootSuiteIds = useSelector((state: State) => state.tree.suites.allRootIds);
```

Comment:
nit: it would be nice to make our own `useSelector` like this:

```ts
import {useSelector as useSelectorBase} from 'react-redux';

export const useSelector<T> = useSelectorBase as (cb: ((state: State) => T) => T);
```
So here you wouldn't need to specify state type:
```ts
const rootSuiteIds = useSelector(state => state.tree.suites.allRootIds);
```

Reply:
Well in fact you can augment default root state without any additional hooks. Implemented it now, so by default useSelector is aware of our types.